Source/Krypton Toolkit/Examples/Classes/DataGridViewSetup.cs

```diff
@@ -1,8 +1,4 @@
-﻿using System.Collections;
-using System.Xml.Linq;
-using System.Xml.XPath;
-
-using Krypton.Toolkit.Suite.Extended.Outlook.Grid;
+﻿//using Krypton.Toolkit.Suite.Extended.Outlook.Grid;
 #pragma warning disable CS8600
```

Comment:
Please state What these are in following comments
Or, remove and fix what they are masking

Reply:
Removed these